dashboard/app/models/contact_rollups_processed.rb

```diff
@@ -157,6 +158,14 @@ def self.extract_user_id(contact_data)
     return values.nil? ? {} : {user_id: values.first}
   end
 
+  def self.extract_professional_learning_enrolled(contact_data)
+    values = extract_field(contact_data, 'dashboard.pd_enrollments', 'course')
+    # We only care about unique and non-nil values.
+    # The result is sorted to keep consistent order.
+    uniq_values = values&.uniq&.compact&.sort
```

Comment:
We already get unique values in the extraction query I think, but I guess can't hurt?

Reply:
yes, it is an extra-safe step for the processing step so it doesn't have to (implicitly) assume that the extraction step does thing in a certain way. Even if the extraction step changes (or introduces a bug), it won't affect the processing step.